### ISSUE TITLE
Add FB opengraph tags to themes

### ISSUE THREAD

**Issue by benjaminjkraft** [learning-unlimited/ESP-Website#2237]
These control how e.g. Facebook, Twitter, and Slack show links to your website.  Several chapters have added them via template overrides, but overriding `elements/html` is kinda asking for trouble and it would be nice if we made this easy for people to do without doing that.

**Comment by haripri0109r** [comment]
Hi @willgearty,

I’d be interested in working on this.

Before starting, should the OpenGraph tags be added to the base theme template (e.g., base.html) with sensible defaults, and optionally overridden per page? 

I’d like to confirm the preferred approach for sourcing title/description/image values before implementing.

Thanks!

**Comment by willgearty** [comment]
Yes, I think you'd want this in our base template (`elements/html`). It should pull from the theme information to inject the group name, logo, etc.

**Comment by haripri0109r** [comment]
Hi @willgearty !,
 I just wanted to follow up on this PR.
Please let me know if any improvements or changes are needed from my side.
Thanks!

**Comment by willgearty** [comment]
@haripri0109r please address copilot comments in the PR: #4418
